src/components/[guild]/EditGuild/components/Events/EventInput.tsx

```diff
@@ -0,0 +1,58 @@
+import {
+  Circle,
+  CloseButton,
+  Img,
+  Input,
+  InputGroup,
+  InputLeftElement,
+  InputRightElement,
+} from "@chakra-ui/react"
+import { useFormContext } from "react-hook-form"
+import { EventSourcesKey, GuildFormType } from "types"
+
+type Props = {
+  name: EventSourcesKey
```

Comment:
I'd call this prop `eventSource`, so we know exactly what's this (it wasn't clear when I first saw this component)